STARTERKIT/gulpfile.js

```diff
@@ -45,6 +49,28 @@ options.theme = {
 // allow Browser Sync to serve the website and update CSS changes on the fly.
 options.drupalURL = '';
 
+// Return true if the given level is equal to or more severe than
+// the configured fatality error level.
+function isFatal(level) {
+  return ERROR_LEVELS.indexOf(level) <= ERROR_LEVELS.indexOf(fatalLevel || 'error');
+}
+
+// Convenience handler for error-level errors.
+function onError(error) {
```

Comment:
I think we can include the code for the function `handleError` inside the `onError` function, right?
as you are changing it's scope, but doesn't seems like it's required.